tests/Mail/MailMailerTest.php

```diff
@@ -91,9 +92,9 @@ public function testMailerCanQueueMessagesToItselfOnAnotherQueue()
     public function testMailerCanQueueMessagesToItselfWithSerializedClosures()
     {
         list($view, $swift) = $this->getMocks();
-        $mailer = new Illuminate\Mail\Mailer($view, $swift);
+        $mailer = new MailerWithFasterSerializer($view, $swift);
```

Comment:
Please don't edit this test. Add a new one because you've basically just deleted the original test here.